server/channels/app/plugin_api_test.go

```diff
@@ -2402,53 +2413,111 @@ func TestPluginServeMetrics(t *testing.T) {
 	require.Equal(t, "METRICS SUBPATH", string(body))
 }
 
-func TestPluginUpdateChannelMembersNotifications(t *testing.T) {
-	t.Run("using API directly", func(t *testing.T) {
```

Comment:
This makes the diff harder to read than I wanted, but I removed this test case because it was duplicated by the "using plugin" case below